Project.toml

```diff
@@ -1,26 +1,26 @@
 name = "ProjectManagement"
 uuid = "8d6fe0f5-cdf5-47ae-8203-553721bd1e97"
 authors = ["Lyndon White"]
-version = "1.1.0"
+version = "1.2.0"
 
 [deps]
 Compose = "a81c6b42-2e10-5240-aca2-a61377ecd94b"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Gadfly = "c91e804a-d5a3-530f-b6f0-dfbca275c004"
+Graphs = "86223c79-3864-5bf0-83f7-82e725a168b6"
 LayeredLayouts = "f4a74d36-062a-4d48-97cd-1356bad1de4e"
-LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 Compose = "0.9"
 DataStructures = "0.17, 0.18"
-Distributions = "0.23"
+Distributions = "0.23, 0.24, 0.25"
+Graphs = "1.5, 1.6"
```

Comment:
This is the same as
```suggestion
Graphs = "1.5"
```

Reply:
right! this is the same. Thanks for reviewing the PR.